user/yaml/kubestellar-server-ingress.yaml

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubestellar
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kubestellar-pvc
+  namespace: kubestellar
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubestellar-service-account
+  namespace: kubestellar
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: kubestellar-role
+  namespace: kubestellar
+rules:
+- resources: [ "secrets" ]
+  verbs: [ "create" , "delete" ]
+  apiGroups: [ "" ]
+- resources: [ "ingresses" ]
+  verbs: [ "get" ]
+  apiGroups: [ "networking.k8s.io" ]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kubestellar-role-binding
+  namespace: kubestellar
+subjects:
+- kind: ServiceAccount
+  name: kubestellar-service-account
+  namespace: kubestellar
+  apiGroup: ""
+roleRef:
+  kind: Role
+  name: kubestellar-role
+  apiGroup: ""
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubestellar-server
+  namespace: kubestellar
+spec:
+  selector:
+    matchLabels:
+      app: kubestellar-server
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: kubestellar-server
+    spec:
+      serviceAccountName: kubestellar-service-account
+      containers:
+      - name: kubestellar-server
+        image: quay.io/kubestellar/kubestellar:fs # for testing, change to :stable

```

Comment:
This creates a version consistency problem.  This YAML is designed for some particular image behavior.

Reply:
I just pushed a stable version, and replaced